lab1.js

```diff
@@ -169,7 +169,7 @@ for (var i = 0; i < mealsPerDay.length ; i++)
   average = sum / (i + 1);
   if (average < 4) {
     tooHungryDay = i + 1;
-    //break; This made the assert fail.
+    break; This made the assert fail.
```

Comment:
yes. Make sure you take out that "this made the assert fail" bit,  because now your file is throwing an error, since that's not accepted syntax for JS.